pkg/teams/teams_test.go

```diff
@@ -0,0 +1,45 @@
+package teams
+
+import "testing"
+
+// This is an example of map that we create from the JSON team map.
+var exampleTeamMap = Teams{
+	Teams: []Team{
+		{
+			Name:     "Test Team 1",
+			Accounts: []string{"000000000001", "000000000011"},
+		},
+		{
+			Name:     "Test Team 2",
+			Accounts: []string{"000000000002", "000000000022"},
+		},
+	},
+}
+
+// This is a helper function to compare two Teams structs and
+// make sure they are identical.
+func compareTeamMaps(a, b Teams) bool {
+	for teamIndex, team := range a.Teams {
+		if team.Name != b.Teams[teamIndex].Name {
+			return false
+		}
+		for acctIndex, acct := range team.Accounts {
+			if acct != b.Teams[teamIndex].Accounts[acctIndex] {
+				return false
+			}
+		}
+	}
+
+	return true
+}
+
+func TestReadTeamMap(t *testing.T) {
+	extractedTeamMap, err := ReadTeamMap("team_map_test.json")
```

Comment:
I would think that the thing you would want to test is the generated maps, not the deserialized struct